conference.js

```diff
@@ -815,41 +815,34 @@ export default {
      * @param {boolean} [showUI] when set to false will not display any error
      * dialogs in case of media permissions error.
      */
-    async mutePresenterVideo(mute, showUI = true) {
+    async mutePresenter(mute, showUI = true) {
         const maybeShowErrorDialog = error => {
             showUI && APP.store.dispatch(notifyCameraError(error));
         };
 
         if (mute) {
             try {
                 await this.localVideo.setEffect(undefined);
-                APP.store.dispatch(
-                    setVideoMuted(mute, MEDIA_TYPE.PRESENTER));
             } catch (err) {
-                logger.error('Failed to mute the Presenter video');
+                logger.error('Failed to remove the presenter effect', err);
+                maybeShowErrorDialog(err);
             }
+        } else {
+            let effect;
 
-            return;
-        }
-        const { height } = this.localVideo.track.getSettings();
-        const defaultCamera
-            = getUserSelectedCameraDeviceId(APP.store.getState());
-        let effect;
-
-        try {
-            effect = await this._createPresenterStreamEffect(height,
-                defaultCamera);
-        } catch (err) {
-            logger.error('Failed to unmute Presenter Video');
-            maybeShowErrorDialog(err);
+            try {
+                effect = await this._createPresenterStreamEffect();
```

Comment:
These 2 blocks can be combined into one:

```
await this.localVideo.setEffect(await this._createPresenterStreamEffect());
```

Reply:
Done